blueprints/app/files/tests/helpers/start-app.js

```diff
@@ -3,16 +3,13 @@ import Application from '../../app';
 import config from '../../config/environment';
 
 export default function startApp(attrs) {
-  let application;
-
   // use defaults, but you can override
-  let attributes = Ember.assign({}, config.APP, attrs);
+  const attributes = Ember.assign({}, config.APP, attrs);
```

Comment:
Hmm, while I disagree with the author's conclusion (he seems to be missing the point of constant *identifiers* (not values) and catering to common confusion instead of addressing it. [Read this for more in the context of identifier reassignment intent](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)), I appreciate the reference 👍